<?xml version="1.0" encoding="utf-8"?>
<TestRun id="f6ccbbf5-d199-4401-b5b7-0891c9b5ca52" name="@14f837b594e6 2022-05-01 07:18:53" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2022-05-01T07:18:53.6014008+00:00" queuing="2022-05-01T07:18:53.6014009+00:00" start="2022-05-01T07:18:52.4173922+00:00" finish="2022-05-01T07:18:53.6074244+00:00" />
  <TestSettings name="default" id="87f132c1-701a-4acd-a0ed-e05267647607">
    <Deployment runDeploymentRoot="_14f837b594e6_2022-05-01_07_18_53" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="2f77deff-f56c-4256-aa04-917023a176f0" testId="08421ada-b90e-b084-333c-821785aa3c19" testName="SqlLocalDataTests.CreateTest.execute_store_procedure" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5185444+00:00" endTime="2022-05-01T07:18:53.5185444+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="2f77deff-f56c-4256-aa04-917023a176f0">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at SqlLocalDataTests.CreateTest.CreateInstance() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 101
   at SqlLocalDataTests.CreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 20
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="1a149de5-67f1-48e5-aaed-d4d4c95851cb" testId="fa4531bc-1afa-7688-8e76-64c004a177f4" testName="SqlLocalDataTests.CreateTest.execute_raw_sql" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5143136+00:00" endTime="2022-05-01T07:18:53.5143250+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="1a149de5-67f1-48e5-aaed-d4d4c95851cb">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at SqlLocalDataTests.CreateTest.CreateInstance() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 101
   at SqlLocalDataTests.CreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 20
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="5a38110c-ccba-4f88-b783-4f45a3756147" testId="1c124193-22f4-325f-434d-3580f2d2849e" testName="SqlLocalDataTests.CreateTest.create_table" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5179008+00:00" endTime="2022-05-01T07:18:53.5179008+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="5a38110c-ccba-4f88-b783-4f45a3756147">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at SqlLocalDataTests.CreateTest.CreateInstance() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 101
   at SqlLocalDataTests.CreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 20
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="ff79f7c5-b962-41ba-8fe3-572f084c5c9d" testId="75682385-a087-96ee-fa56-e686030901b9" testName="SqlLocalDataTests.FastCreateTest.execute_raw_sql" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5176561+00:00" endTime="2022-05-01T07:18:53.5176561+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="ff79f7c5-b962-41ba-8fe3-572f084c5c9d">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 178
   at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 74
   at SqlLocalDataTests.FastCreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 22
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="7e30a7a5-7af5-460b-a361-46924d7d2e06" testId="b3371dc3-18f8-8510-8676-34f37e7b3c22" testName="SqlLocalDataTests.FastCreateTest.execute_store_procedure" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5181003+00:00" endTime="2022-05-01T07:18:53.5181004+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="7e30a7a5-7af5-460b-a361-46924d7d2e06">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 178
   at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 74
   at SqlLocalDataTests.FastCreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 22
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="3ac238e7-b5cf-4b0b-b478-13add91e3948" testId="105e5502-47be-b06e-df1e-965369cfe4b4" testName="SqlLocalDataTests.FastCreateTest.execute_raw_sql_from_file" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5166491+00:00" endTime="2022-05-01T07:18:53.5166491+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3ac238e7-b5cf-4b0b-b478-13add91e3948">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 178
   at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 74
   at SqlLocalDataTests.FastCreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs:line 22
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="2d906e09-db65-4cd2-8628-3b0000787503" testId="ba18da23-2af1-6cbe-e2e9-e21654bd6d35" testName="SqlLocalDataTests.CreateTest.execute_raw_sql_from_file" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5174567+00:00" endTime="2022-05-01T07:18:53.5174568+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="2d906e09-db65-4cd2-8628-3b0000787503">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at SqlLocalDataTests.CreateTest.CreateInstance() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 101
   at SqlLocalDataTests.CreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 20
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="08612a88-2253-4e0f-b665-56f68567e5b1" testId="6ca2b253-3bc0-49a2-0edd-0773739b08d4" testName="SqlLocalDataTests.CreateTest.database_exists" computerName="14f837b594e6" duration="00:00:00.0010000" startTime="2022-05-01T07:18:53.5188082+00:00" endTime="2022-05-01T07:18:53.5188083+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="08612a88-2253-4e0f-b665-56f68567e5b1">
      <Output>
        <ErrorInfo>
          <Message>System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory</Message>
          <StackTrace>   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments) in /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs:line 33
   at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 112
   at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext() in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 122
   at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
   at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName) in /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs:line 151
   at SqlLocalDataTests.CreateTest.CreateInstance() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 101
   at SqlLocalDataTests.CreateTest..ctor() in /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs:line 20
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="SqlLocalDataTests.CreateTest.database_exists" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="6ca2b253-3bc0-49a2-0edd-0773739b08d4">
      <Execution id="08612a88-2253-4e0f-b665-56f68567e5b1" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.CreateTest" name="database_exists" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.CreateTest.execute_store_procedure" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="08421ada-b90e-b084-333c-821785aa3c19">
      <Execution id="2f77deff-f56c-4256-aa04-917023a176f0" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.CreateTest" name="execute_store_procedure" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.FastCreateTest.execute_raw_sql_from_file" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="105e5502-47be-b06e-df1e-965369cfe4b4">
      <Execution id="3ac238e7-b5cf-4b0b-b478-13add91e3948" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.FastCreateTest" name="execute_raw_sql_from_file" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.FastCreateTest.execute_store_procedure" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="b3371dc3-18f8-8510-8676-34f37e7b3c22">
      <Execution id="7e30a7a5-7af5-460b-a361-46924d7d2e06" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.FastCreateTest" name="execute_store_procedure" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.CreateTest.execute_raw_sql_from_file" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="ba18da23-2af1-6cbe-e2e9-e21654bd6d35">
      <Execution id="2d906e09-db65-4cd2-8628-3b0000787503" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.CreateTest" name="execute_raw_sql_from_file" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.FastCreateTest.execute_raw_sql" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="75682385-a087-96ee-fa56-e686030901b9">
      <Execution id="ff79f7c5-b962-41ba-8fe3-572f084c5c9d" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.FastCreateTest" name="execute_raw_sql" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.CreateTest.execute_raw_sql" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="fa4531bc-1afa-7688-8e76-64c004a177f4">
      <Execution id="1a149de5-67f1-48e5-aaed-d4d4c95851cb" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.CreateTest" name="execute_raw_sql" />
    </UnitTest>
    <UnitTest name="SqlLocalDataTests.CreateTest.create_table" storage="/sqllocaldatatests/sqllocaldatatests/bin/debug/net6.0/sqllocaldatatests.dll" id="1c124193-22f4-325f-434d-3580f2d2849e">
      <Execution id="5a38110c-ccba-4f88-b783-4f45a3756147" />
      <TestMethod codeBase="/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0/SqlLocalDataTests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="SqlLocalDataTests.CreateTest" name="create_table" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="08421ada-b90e-b084-333c-821785aa3c19" executionId="2f77deff-f56c-4256-aa04-917023a176f0" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="fa4531bc-1afa-7688-8e76-64c004a177f4" executionId="1a149de5-67f1-48e5-aaed-d4d4c95851cb" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="1c124193-22f4-325f-434d-3580f2d2849e" executionId="5a38110c-ccba-4f88-b783-4f45a3756147" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="75682385-a087-96ee-fa56-e686030901b9" executionId="ff79f7c5-b962-41ba-8fe3-572f084c5c9d" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="b3371dc3-18f8-8510-8676-34f37e7b3c22" executionId="7e30a7a5-7af5-460b-a361-46924d7d2e06" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="105e5502-47be-b06e-df1e-965369cfe4b4" executionId="3ac238e7-b5cf-4b0b-b478-13add91e3948" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="ba18da23-2af1-6cbe-e2e9-e21654bd6d35" executionId="2d906e09-db65-4cd2-8628-3b0000787503" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="6ca2b253-3bc0-49a2-0edd-0773739b08d4" executionId="08612a88-2253-4e0f-b665-56f68567e5b1" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="8" executed="8" passed="0" failed="8" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.4.3+1b45f5407b (64-bit .NET 6.0.4)
[xUnit.net 00:00:00.67]   Discovering: SqlLocalDataTests
[xUnit.net 00:00:00.69]   Discovered:  SqlLocalDataTests
[xUnit.net 00:00:00.69]   Starting:    SqlLocalDataTests
[xUnit.net 00:00:00.74]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.74]       Stack Trace:
[xUnit.net 00:00:00.74]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.74]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.74]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.74]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(101,0): at SqlLocalDataTests.CreateTest.CreateInstance()
[xUnit.net 00:00:00.74]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(20,0): at SqlLocalDataTests.CreateTest..ctor()
[xUnit.net 00:00:00.74]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(178,0): at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(74,0): at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(22,0): at SqlLocalDataTests.FastCreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(101,0): at SqlLocalDataTests.CreateTest.CreateInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(20,0): at SqlLocalDataTests.CreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(178,0): at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(74,0): at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(22,0): at SqlLocalDataTests.FastCreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(101,0): at SqlLocalDataTests.CreateTest.CreateInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(20,0): at SqlLocalDataTests.CreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(178,0): at T1.SqlLocalData.SqlLocalDb.EnsureInstanceCreated(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(74,0): at SqlLocalDataTests.FastCreateTest.InitializeSqlLocalDbInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/FastCreateTest.cs(22,0): at SqlLocalDataTests.FastCreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(101,0): at SqlLocalDataTests.CreateTest.CreateInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(20,0): at SqlLocalDataTests.CreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]       System.ComponentModel.Win32Exception : An error occurred trying to start process '/opt/mssql-tools/bin' with working directory '/SqlLocalDataTests/SqlLocalDataTests/bin/Debug/net6.0'. No such file or directory
[xUnit.net 00:00:00.75]       Stack Trace:
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32&amp; stdinFd, Int32&amp; stdoutFd, Int32&amp; stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
[xUnit.net 00:00:00.75]            at System.Diagnostics.Process.Start()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/ProcessHelper.cs(33,0): at T1.SqlLocalData.ProcessHelper.Execute(String processFilename, String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(112,0): at T1.SqlLocalData.SqlLocalDb.ExecuteSqlLocalDbExe(String arguments)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(122,0): at T1.SqlLocalData.SqlLocalDb.GetInstanceNames()+MoveNext()
[xUnit.net 00:00:00.75]            at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/T1.SqlLocalData/SqlLocalDb.cs(151,0): at T1.SqlLocalData.SqlLocalDb.IsInstanceExists(String instanceName)
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(101,0): at SqlLocalDataTests.CreateTest.CreateInstance()
[xUnit.net 00:00:00.75]         /SqlLocalDataTests/SqlLocalDataTests/CreateTest.cs(20,0): at SqlLocalDataTests.CreateTest..ctor()
[xUnit.net 00:00:00.75]            at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[xUnit.net 00:00:00.75]   Finished:    SqlLocalDataTests
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5108232+00:00">
        <Text>[xUnit.net 00:00:00.74]     SqlLocalDataTests.CreateTest.execute_raw_sql [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5165058+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.FastCreateTest.execute_raw_sql_from_file [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5174422+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.CreateTest.execute_raw_sql_from_file [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5177401+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.FastCreateTest.execute_raw_sql [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5182389+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.CreateTest.create_table [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5186992+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.FastCreateTest.execute_store_procedure [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5192680+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.CreateTest.execute_store_procedure [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="14f837b594e6" outcome="Error" timestamp="2022-05-01T07:18:53.5200485+00:00">
        <Text>[xUnit.net 00:00:00.75]     SqlLocalDataTests.CreateTest.database_exists [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>